src/libraries/System.Private.CoreLib/src/System/DateTime.cs

```diff
@@ -625,6 +1153,19 @@ internal static ulong TimeToTicks(int hour, int minute, int second, int millisec
             return ticks;
         }
 
+        internal static ulong TimeToTicks(int hour, int minute, int second, int millisecond, int microsecond)
+        {
+            ulong ticks = TimeToTicks(hour, minute, second, millisecond);
+
+            if ((uint)microsecond >= MicrosecondsPerMillisecond) ThrowMicrosecondOutOfRange();
+
+            ticks += (uint)microsecond * (uint)TicksPerMicrosecond;
```

Comment:
>uint [](http://example.com/codeflow?start=22&length=4)

do we need the casting in this line?

Reply:
See https://github.com/dotnet/runtime/pull/67666#discussion_r847556668